README.md

```diff
@@ -1,94 +1,87 @@
+||||
+|:point_right:| **Make Juju your own!** | Write your own charm: <br> https://juju.is/docs/sdk <br> _Juju is written in Go, but our SDK supports easy charm development in Python!_ |
 
-<!-- The text below is also at the top of ops/__init__.py. Keep in sync! -->
+## Use Ops
+Give it a quick try:
 
-The ops library is a Python framework ([`available on PyPI`](https://pypi.org/project/ops/)) for developing
-and testing [Juju](https://juju.is/) charms in a consistent way, using standard Python constructs
-to allow for clean, maintainable, and reusable code.
+_(assumes you have a Linux that supports installation via snap)_
 
-A charm is an operator -- business logic encapsulated in a reusable software
-package that automates every aspect of an application's life.
-
-Charms written with ops support Kubernetes using Juju's "sidecar charm"
-pattern, as well as charms that deploy to Linux-based machines and containers.
-
-Charms should do one thing and do it well. Each charm drives a single
-application and can be integrated with other charms to deliver a complex
-system. A charm handles creating the application in addition to scaling,
-configuration, optimisation, networking, service mesh, observability, and other
-day-2 operations specific to the application.
-
-The ops library is part of the Charm SDK (the other part being Charmcraft).
-Full developer documentation for the Charm SDK is available at
-https://juju.is/docs/sdk.
-
-To learn more about Juju, visit https://juju.is/docs/olm.
-
-
-## Pure Python
+```
+# Use the Multipass 'charm-dev' blueprint to quickly get an Ubuntu VM with Charmcraft
+# as well as everything else you’ll need to develop and test a machine/ Kubernetes charm,
+# including `tox`, a LXD / Kubernetes cloud -- 'localhost' / 'microk8s' -- and Juju,
+# then use Juju to bootstrap a controller into a cloud -- below, 'microk8s'-
+# and add a model that you'll deploy your charm to.
+sudo snap install multipass && \
+multipass launch --cpus 4 --memory 8G --disk 30G --name tutorial-vm charm-dev
+juju bootstrap microk8s tutorial-controller; juju add-model tutorial-model
 
-The framework provides a standardised Python object model that represents the
-application graph, as well as an event-handling mechanism for distributed
-system coordination and communication.
+# Create a charm directory and use Charmcraft to initialise your charm file structure:
+mkdir my-new-charm; cd my-new-charm; charmcraft init
 
-The latest version of ops requires Python 3.8 or above.
```

Comment:
Why drop this? As a Python dev, I don't want to go digging through pyproject.toml or setup.py to find this information, and clicking through to PyPI just to get it takes me away from what I'm reading.

Reply:
Happy to work it into the intro paragraph.